posts/2023-01-09-android-ndk-update-r25.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: "Updating the Android NDK in Rust <version>"
+author: Android Platform Team
+description: "Modernizing Android support in Rust"
+---
+
+We are pleased to announce that Android platform support in Rust will be
+modernized in Rust 1.66 as we update the target NDK from r17 to r25.  As a
```

Comment:
```suggestion
modernized in Rust 1.68 as we update the target NDK from r17 to r25.  As a
```

Presuming we expect to land this prior to January 20th, which is the cut-off for 1.68; otherwise 1.69 is the right thing here. Not sure if we will hit the 20th; I think an alternative is writing 1.68 or 1.69 here.